app/serializers/disability_claim_base_serializer.rb

```diff
@@ -5,6 +5,10 @@ class DisabilityClaimBaseSerializer < ActiveModel::Serializer
              :development_letter_sent, :decision_letter_sent, :successful_sync,
              :updated_at, :phase
 
+  def id
+    object.evss_id
+  end
```

Comment:
Do we want to remove `evss_id` from the attributes now? Or just leave that as a todo so there's less front end churn? If we leave it, I think we should add a comment so we don't forget why there are two identical IDs in the response.